src/index.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import { BrowserRouter as Routes } from 'react-router-dom';
+import BookStore from './BookStore';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Routes>
+      <BookStore />
+    </Routes>
   </React.StrictMode>,
 );
```

Comment:
- [ ] `<Routes>` is already present in the BookStore component. As far as I can see, you've put `<Routes>` inside another `<Routes>`. Please remove one so that your code is simple and DRY.

Reply:
@medaminedev66 Oo  it's an alias for BrowserRouter